### ISSUE TITLE
Coral will not stay planted in Colored Water

### ISSUE THREAD

**Issue by Sharpy84** [BillyGalbreath/ColoredWater#2]
BiomesOPlenty coral will not stay planted when in ColoredWater, I am unsure who to report this to, BiomesOPlenty or ColoredWater so i am reporting to both, please feel free to close if this isn't your issue.

**Comment by BillyGalbreath** [comment]
I don't believe it's my issue. These water blocks are extensions of vanilla water.

I will leave this open to see what BoP has to say. I linked them to the source since they didn't find it and closed your ticket
